src/components/GameBoard.vue

```diff
@@ -23,7 +23,7 @@
     </div>
 </template>
 <script>
-const MAX_BAD_GUESSES = 10
+const MAX_BAD_GUESSES = 18
```

Comment:
Wow, yall are too nice!

Reply:
I hadn't realized that side-effect. That just happened to be how many parts of snowman were in that svg.

Expect Bryce to add a word generator feature at some point so that the host can play along. Maybe a difficulty setting would allow the user to set the bad guesses?

Reply:
Don't merge this yet, please. It is too easy so I'll make an update.